examples/config.json

```diff
@@ -28,19 +28,29 @@
         "vector_tile_dragndrop": "Drag and drop a style"
     },
 
-    "Source / Stream": {
+    "WFS source": {
         "source_stream_wfs_25d": "WFS to 2.5D objects",
         "source_stream_wfs_3d": "WFS to 3D objects",
         "source_stream_wfs_raster": "WFS to raster"
     },
 
-    "Source / File": {
-        "source_file_gpx_3d": "GPX to 3D objects",
-        "source_file_geojson_3d": "GeoJSON to 3D objects",
-        "source_file_shapefile": "Shapefile to raster",
+    "Specific source options": {
+        "source_file_from_format": "format",
+        "source_file_from_methods": "fetcher and parser"
+    },
+
+    "Customize FileSource": {
+        "source_file_from_fetched_data": "Handle fetching step",
+        "source_file_from_parsed_data": "Handle parsing step"
+    },
+
+    "FileSource": {
         "source_file_geojson_raster": "GeoJSON to raster",
+        "source_file_geojson_3d": "GeoJSON to 3D objects",
+        "source_file_kml_raster": "KML to raster",
         "source_file_gpx_raster": "GPX to raster",
-        "source_file_kml_raster": "KML to raster"
+        "source_file_gpx_3d": "GPX to 3D objects",
+        "source_file_shapefile": "Shapefile to raster"
```

Comment:
source_file_shapefile move to `Customize FileSource` and rename to `shape file case`

 "Customize FileSource" move after  "FileSource"

Reply:
I changed the config accordingly.